### PR TITLE
Dev/project exercise 4

### PR DESCRIPTION
**Pull request by CraigUffindall**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: my-ansible-playbook.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: present
```
Comment (hugh-emerson) [inline my-ansible-playbook.yml:25]: I think this task ought to have `become: yes` as well, like the "install git" task.

File: my-ansible-playbook.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3 -
+      creates: /usr/local/bin
```
Comment (hugh-emerson) [inline my-ansible-playbook.yml]: The purpose of `creates` on the shell task is to say "this task should be skipped if <file> already exists".

So it should point to the poetry file. If it's something general like /usr/local/bin, then it's always true that it exists so this task will always get skipped.

**Review by hugh-emerson — APPROVED** [review]
Nice work 👍 Some small comments but go ahead and merge this

File: my-ansible-playbook.yml (hunk 1 above)
Comment (CraigUffindall) [reply my-ansible-playbook.yml:25]: Added become: yes as suggested

File: my-ansible-playbook.yml (hunk 2 above)
Comment (CraigUffindall) [reply my-ansible-playbook.yml]: Changed to point to the poetry file.

**Pull request merged**